examples/webgl_loader_mmd.html

```diff
@@ -160,23 +172,76 @@
 
 				if( mesh ) {
 
-					mixer.update( clock.getDelta() );
-
-					/* Note: ikSolver.resetLinks() is workaround for r74dev.
-					 *       THREE.PropertyMixer.apply() seems to have buffer cache
-					 *       when mixer.update() is called.
-					 *       ikSolver.update() change bone position/quaternion
-					 *       without mixer.update() then buffer cache will be inconsistent.
-					 *       ikSolver.resetLinks() explicitly resets IK link bones quaternion and
-					 *       tries to avoid issues caused by this inconsistent.
-					 */
-					ikSolver.resetLinks();
+					var delta = clock.getDelta();
+					mixer.update( delta );
+
+					// see below for what this's for
+					backupBones( mesh );
+
 					ikSolver.update();
+					physics.update( delta );
 
 				}
 
 				renderer.render( scene, camera );
 
+				if ( mesh ) {
+
+					// see below for what this's for
+					restoreBones( mesh );
+
+				}
+
+			}
+
+			/*
+			 * Note: These following three functions are workaround for r74dev.
+			 *       THREE.PropertyMixer.apply() seems to save values into buffer cache
```

Comment:
Yes. The right order is mixer, ikSolver, then physics

	var delta = clock.getDelta();
	mixer.update( delta ); // mixer is an instance of THREE.AnimationMixer
	ikSolver.update(); // directly(without bindings) changing bone position/quaternion
	physics.update( delta );  // directly(without bindings) changing bone position/quaternion

The bones addressed by IK and Physics should be pre-specified.
After skinning all bones, IK and Physics calculate
the final position/quaternion from skinned bones for them in each frame.
My IK uses a generic algorithm CCD-IK so could be reusable
but physics may be MMD specific.

I'm not quite sure the detail of Three.js animation system inside so
can't say yet if it's operatable on PropertyMixers.

I think there'd better to have a way to add user-plugin for mixer
because there're some animation techniques which directly address
the bones like these IK and Physics.